satellite/audit/observer.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/satellite/overlay"
+)
+
+// Observer observes on the metainfo loop and adds segments to node reservoirs
+type Observer struct {
+	log *zap.Logger
+
+	overlay        *overlay.Service
```

Comment:
why do you need the overlay to be here? It doesn't look like it is being used

Reply:
oh shoot- took it out when I no longer needed to check if nodes are vetted. I'll remove it.